a10_octavia/controller/worker/tasks/a10_network_tasks.py

```diff
@@ -58,21 +58,29 @@ class CalculateAmphoraDelta(BaseNetworkTask):
 
     default_provides = constants.DELTA
 
-    def execute(self, loadbalancer, amphora):
+    def execute(self, loadbalancers_list, amphora):
         LOG.debug("Calculating network delta for amphora id: %s", amphora.id)
         # Figure out what networks we want
         # seed with lb network(s)
-        vrrp_port = self.network_driver.get_port(amphora.vrrp_port_id)
-        desired_network_ids = {vrrp_port.network_id}.union(
-            CONF.controller_worker.amp_boot_network_list)
-
-        for pool in loadbalancer.pools:
-            member_networks = [
-                self.network_driver.get_subnet(member.subnet_id).network_id
-                for member in pool.members
-                if member.subnet_id
-            ]
-            desired_network_ids.update(member_networks)
+
+        desired_network_ids = set()
+        desired_network_ids.update(CONF.controller_worker.amp_boot_network_list)
```

Comment:
Is there any issue when union with amphora.vrrp_port_id?

Reply:
`amphora.vrrp_port_id` is the loadbalancer's vip subnet-id. 
So in case of loadbalancer deletion, 
`desired_network_ids = {vrrp_port.network_id}.union(CONF.controller_worker.amp_boot_network_list)` is adding the loadbalancer's vip subnet-id and lb-mgmt subnet-id to `desired_network_ids`.
And due to this, del_ids is becoming empty in the loadbalancer deletion case and so providing nothing in delete_nics.
As a result of this, evenif we want to delete the interface attached to the deleting LB by getting network-id of currently deleting LB in del_ids, we will not be able to do so as `actual_network_nics == desired_network_nics`.
So to avoid this case, removed that union part.

Reply:
OK, I think this should be fine since:
- vThunder can use any interface to do vrrp-a sync. not only vip subnet.
- When the vrrp-a interface is deattached, vthunders need to reload/reboot. After reboot/reload they will find another interface to do vrrp-a sync. 
- And if the vrrp-a interface is the latest interface, then vthunder will be deleted after vport is deleted. so, no vrrp-a sync. problem.